tutoroars/plugin.py

```diff
@@ -43,6 +45,9 @@
                 "edx-event-routing-backends==5.3.1",
             ],
         ),
+        # Vector settings
+        ("VECTOR_DOCKER_IMAGE", "docker.io/timberio/vector:0.30.0-alpine"),
```

Comment:
This is a version bump from what Cairn uses so we could get the `date_time_best_effort` setting in the ClickHouse sink to simplify the parsing of more exact dates.